src/main/java/io/vertx/core/Context.java

```diff
@@ -100,6 +101,32 @@ static boolean isOnVertxThread() {
   void runOnContext(Handler<Void> action);
 
   /**
+   * Like {@link #executeBlocking(Handler, boolean, Handler)} called with ordered = true.
+   */
+  <T> void executeBlocking(Action<T> action, Handler<AsyncResult<T>> resultHandler);
```

Comment:
Action is not a public class - this method should not be part of the public API.